NEWS

```diff
@@ -57,7 +57,39 @@ included in the vX.Y.Z section and be denoted as:
 --------------------
 
 Bug fixes/minor improvements:
+
 - Remove IB XRC support from the OpenIB BTL due to lack of support.
+- Fix a problem with MPI_IALLTOALLW when using zero-length messages.
```

Comment:
For the XRC bullet, it says "support" twice -- perhaps it can be:

> Remove IB XRC functionality from the OpenIB BTL due to lack of support.

...the more I think about this, though,  are we breaking backwards compatibility?  I.e., is the broken XRC functionality a regression since v2.0.0?  Or has it been 100% broken since v2.0.0?  (i.e., it can't break backwards compatibility because there's no possible chance that anyone has been using it because it has been completely broken)